File: CierreDeCajaBancosDax/ResumenCajaGeneral -or/DataSet1.cs
using System.Data;

namespace CierreDeCaja
{
    public partial class DataSet1
    {
        partial class DataTableGenDataTable
        {
        }

        public partial class DataTable2DataTable
        {
            private void DataTable2DataTable_ColumnChanging(object sender, DataColumnChangeEventArgs e)
            {
                if ((e.Column.ColumnName ?? "") == (UsuarioColumn.ColumnName ?? ""))
                {
                    // Agregar código de usuario aquí
                }
            }
        }

        public partial class DataTable1DataTable
        {
        }
    }
}

File: CierreDeCajaBancosDax/CierreCajaDax/DataSet1.vb


Partial Public Class DataSet1
    Partial Public Class DataTable2DataTable
        Private Sub DataTable2DataTable_ColumnChanging(sender As Object, e As DataColumnChangeEventArgs) Handles Me.ColumnChanging
            If (e.Column.ColumnName = Me.UsuarioColumn.ColumnName) Then
                'Agregar código de usuario aquí
            End If

        End Sub

    End Class

    Partial Class DataTable1DataTable

    End Class

End Class
